tests/testthat/test_rt_login.R

```diff
@@ -0,0 +1,5 @@
+context("Test login")
+
+test_that("A bad username/password returns an error", {
```

Comment:
Great idea to add a test. However, this test is kind of brittle: It depends upon our NCEAS RT system which may not be a good idea because (1) we may take that service down, at which point this test needs to get fixed and (2) since Travis runs out tests every commit/pr, we're hitting our RT instance with bogus login attempts (plus exposing our RT instance publicly).

This test really tests:

- Whether our RT system is up
- Whether it's deployed at the same place as it is now
- Whether a user with the username 'user' and 'pass' exists on the system
- Whether the test environment has a connection to the Internet that's reliable

Where you really want to test whether your function correctly stops when the response includes the text "Credentials required". To test that functionality, you could extract that logic into a separate function and test the behavior directly.

Reply:
Hmm good points. In the back of my mind, I thought that maybe the RT response might change over time, and we'd want to be able to catch it if it does.  I presumed that the way you originally wrote the function worked at the time that you wrote it.  

If I boil it down, it seems like I'd really just be testing to see if I wrote the `str_detect` for "Credentials required" correctly...so perhaps I should just remove the test altogether?  Further thoughts?

Reply:
> I thought that maybe the RT response might change over time, and we'd want to be able to catch it if it does